apps/dapp/src/components/Profile/Edit/EditProfileForAccount.tsx

```diff
@@ -75,6 +88,16 @@ export default function EditProfileForAccount({
     [isUpdateProfilePending, isUpdatingData],
   );
 
+  useEffect(() => {
```

Comment:
NIT: I'm seeing this block of code a lot. We can move this to a new hook and do something like:

```tsx
export interface WithError {
  message: string;
}

export const useToastError = (withError: WithError | undefined, title?: string) => {
  const { toast } = useToast();
  
  useEffect(() => {
     toast({
        title: title || 'Error',
        description: withError.message,
        variant: 'error',
      });
    }, [withError, title]);
} 
```

Then use it like this:

```tsx
const { uploadProfileData, uploadProfileDataError } = useUploadProfileData();
useToastError(uploadProfileDataError, 'Upload Error');
```

Reply:
Done